test/functional/core/job_spec.lua

```diff
@@ -274,11 +272,7 @@ describe('jobs', function()
 
   it('can omit options', function()
     neq(0, nvim('eval', 'delete(".Xtestjob")'))
-    local touch_cmd = iswin() and "'Out-File -encoding ASCII .Xtestjob'"
-                              or "['touch', '.Xtestjob']"
-    nvim('command', "call jobstart(" .. touch_cmd .. ")")
-    nvim('command', "sleep " .. (iswin() and "5" or "100m"))
-    eq(0, nvim('eval', 'delete(".Xtestjob")'))
+    eq(1, eval("jobstart(['nvim', '-u', 'NONE', '-c', 'q', '.Xtestjob']) > 0"))
```

Comment:
eval returned 0 on the sanitizer build on Travis but this worked in Appveyor. How is that possible?